packages/backend-common/api-report.md

```diff
@@ -491,7 +491,12 @@ export class ServerTokenManager implements TokenManager {
   // (undocumented)
   authenticate(token: string): Promise<void>;
   // (undocumented)
-  static fromConfig(config: Config): ServerTokenManager;
+  static fromConfig(
+    config: Config,
+    options: {
```

Comment:
Do you think it might be worth making the logger (and the options object) optional and therefore making this not a breaking change? (If the logger is not passed in, we just `console.warn` instead) Or is it worth making it a breaking change to force the usage of a logger?

Reply:
:+1: done in 832a4fe1c4.